php/class-hubspot-contacts.php

```diff
@@ -157,6 +149,20 @@ public static function instance() {
 	public function setup() {
 		global $wp;
 
+		// Adding messages here so we can call __().
+		$messages = array(
+			'error' => __( 'Sorry, an error occurred, please try again.', 'hubspot_subscribe' ),
+			'signed_up' => __( 'You\'re already signed up! Please check your inbox for a confirmation message with a link to change your settings.', 'hubspot_subscribe' ),
+			'update_error' => __( 'Failed to update your data, please try again.', 'hubspot_subscribe' ),
+			'signup_error' => __( 'Failed to create your subscription, please try again.', 'hubspot_subscribe' ),
+			'settings_saved' => __( 'Your settings have been saved. <a href="%s">Edit Settings</a>', 'hubspot_subscribe' ),
+			'not_found' => __( 'Your settings could not be found.', 'hubspot_subscribe' ),
+			'opt_out' => __( 'You have opted out of all email subscriptions. To opt back in, use the link in one of the emails you\'ve already received. If you\'ve recently opted back in, please allow 5-10 minutes.', 'hubspot_subscribe' ),
```

Comment:
same as above re:double quotes. this plugin should have travis and phpcs setup, it would catch these things.